signalfx-codahale/src/main/java/com/signalfx/codahale/reporter/AggregateMetricSenderSessionWrapper.java

```diff
@@ -29,7 +32,8 @@
             MetricMetadata metricMetadata,
             String defaultSourceName,
             String sourceDimension) {
-        this(metricSenderSession, detailsToAdd, metricMetadata, defaultSourceName, sourceDimension, false);
+        this(metricSenderSession, detailsToAdd, metricMetadata, defaultSourceName, sourceDimension,
+                false, ImmutableMap.<String, String> of());
```

Comment:
Why not Collections.emptyMap() (Not suggesting, just curious if there is a best practice?)